influxql/engine.go

```diff
@@ -752,6 +752,11 @@ func MapStddev(itr Iterator, e *Emitter, tmax int64) {
 
 	for k, v := itr.Next(); k != 0; k, v = itr.Next() {
 		values = append(values, v.(float64))
+		// Emit in batches of 1000
```

Comment:
If we're not going to make this configurable, perhaps we should at least add a comment explaining *why* we are doing this.